routes/cohortRoutes.js

```diff
@@ -0,0 +1,96 @@
+const express = require('express');
+const router = express.Router();
+
+const knex = require('knex');
+const knexConfig = require('../knexfile');
+const db = knex(knexConfig.development);
+
+const responseStatus = {
+  success: 200,
+  postCreated: 201,
+  badRequest: 400,
+  notFound: 404,
+  serverError: 500
+};
+
+router.get('/', (req, res) => {
+  db('cohorts')
+    .then(cohorts => {
+      res.status(responseStatus.success).json(cohorts);
+    })
+    .catch(err => res.status(responseStatus.serverError).json(err));
+});
+
+router.post('/', async (req, res) => {
+  try {
+    const ids = await db('cohorts').insert(req.body);
+    const cohortResponse = await db('cohorts').where({ id: ids[0] });
+    res.status(responseStatus.postCreated).json(cohortResponse);
```

Comment:
Reading other people's code takes some getting used to because you have to get in their head, but this async/await is so clean. It makes reading your code at least twice as fast as `.then() and .catch()`. Coupled w/ your responseStatus object makes for very clean-readable code.